File: solutions/2025-dec/15/speed_check.py
def speed_check(speedMph, speedLimitKph):
    speedKph = speedMph * 1.60934
    if speedKph <= speedLimitKph:
        return "Not Speeding"
    elif speedKph <= speedLimitKph + 5:
        return "Warning"
    return "Ticket"

File: solutions/2025-dec/15/speed_check.js
function speedCheck(speedMph, speedLimitKph) {
    const speedKph = speedMph * 1.60934;
    if (speedKph <= speedLimitKph) return "Not Speeding";
    else if (speedKph <= speedLimitKph + 5) return "Warning";
    return "Ticket";
}
